style.css

```diff
@@ -0,0 +1,198 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Poppins';
+}
+
+a {
+    text-decoration: none;
+    color: #000;
+}
+h2{
+    font-size:1.7em;
+}
+h3{
+    font-size:0.9em;
+}
+header#header>div {
+    padding: 10px 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+}
+
+.slogan {
+    font-size: 0.8em;
+}
+
+nav.menu {
+    width: 470px;
+}
+
+nav.menu ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 0.8em;
+    font-weight: 600;
+}
+
+nav.menu a {
+    color: black;
+}
+
+.blackheart {
+    width: 20px;
+    margin-right: 1rem;
+}
+
+header#header #topbar-search {
+    padding
+}
+
```

Comment:
Trying to point out the only mistakes I see,
Here you declared a padding but assigned no value, this can lead to bugs as it might be misinterpreted.
You can either comment it out or take it out :)